lib/krane/cli/run_command.rb

```diff
@@ -19,10 +19,11 @@ class RunCommand
         },
         "verify-result" => { type: :boolean, desc: "Wait for completion and verify pod success", default: true },
         "command" => { type: :array, desc: "Override the default command in the container image" },
-        "template" => {
+        "filename" => {
```

Comment:
I thought we decided against this https://github.com/Shopify/kubernetes-deploy/issues/568

Reply:
Good catch, thank you! I wasn't aware of that. I'll update this PR undoing this particular change.

Reply:
Done. This is ready for another review.